packages/reactotron-server/src/server/index.test.ts

```diff
@@ -0,0 +1,36 @@
+import test from "ava"
+import * as request from "request-promise-native"
+import { httpServerInstance, apolloServerInstance } from './'
+
+// Tests in here are ran in serial due to the webserver binding to port 4000 (concurrency results in EADDRINUSE error)
+
+let app, httpServer;
+
+test.serial("Can create a http server to serve react app", async t => {
+
+  let appServer;
+
+  t.notThrows(() => {
+    appServer = httpServerInstance()
+  })
+
+  // Not sure if there is a way to nicely destructure here, this does for now
+  app = appServer.app;
+  httpServer = appServer.httpServer;
+
+  t.true(httpServer !== null && typeof httpServer === "object")
+
+  const responseString = await request("http://localhost:4000").catch(err => {
```

Comment:
Ideally we would make the port option come from config here, but this is a fine first pass and we can update as we update the src.